assets/js/modules/ads/components/setup/SetupMainPAX.stories.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * SetupMainPAX component stories.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependecies.
+ */
+import { withQuery } from '@storybook/addon-queryparams';
+
+/**
+ * Internal dependencies
+ */
+import { ADWORDS_SCOPE, MODULES_ADS } from '../../datastore/constants';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import {
+	provideModuleRegistrations,
+	provideSiteInfo,
+	provideModules,
+	provideUserAuthentication,
+} from '../../../../../../tests/js/utils';
+import SetupMainPAX from './SetupMainPAX';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+
+function Template( { setupRegistry = () => {} } ) {
+	return (
+		<WithRegistrySetup func={ setupRegistry }>
+			<SetupMainPAX />
+		</WithRegistrySetup>
+	);
+}
+
+export const Default = Template.bind( {} );
+Default.storyName = 'Default';
+Default.args = {
+	setupRegistry: ( registry ) => {
+		registry
+			.dispatch( MODULES_ADS )
+			.setSettings( { paxConversionID: 'AW-123456789' } );
+
+		registry.dispatch( CORE_USER ).receiveIsAdBlockerActive( false );
+	},
+};
+Default.scenario = {
+	label: 'Modules/Ads/Setup/SetupMainPAX/Default',
+};
+
+export const withAdwordsScope = Template.bind( {} );
```

Comment:
Since the default state in most cases is the one where person is not going to have the scope, it seems more appropriately for it to be default, and not permission message is shown in story that references it in the name.

> I don't see any Figma mocks in this issue, is there an existing design for this or is the text/design here ad-hoc?

Nope, there is no design or anything :D I can remove the color